CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+# Contributing Guidelines
+
+There are many ways to contribute to CoW Protocol. We welcome contributions of all kinds from anyone, whether you've been a long-time user or you are just getting started with development.
+
+## Troubleshooting
+
+You can help other users in the community to solve their issues in the [CoW Protocol Discord].
+
+[CoW Protocol Discord]: https://discord.gg/cowprotocol
+[CoW Protocol Forums]: https://forum.cow.fi
+
+## Opening an issue
+
+You can [open an issue] to suggest a feature or report a minor bug.
+
+If you believe your issue may be due to user error and not a problem in the library, consider instead posting a question on the [CoW Protocol Discord].
+
+Before opening an issue, be sure to search through the existing open and closed issues, and consider posting a comment in one of those instead.
+
+When requesting a new feature, include as many details as you can, especially around the use cases that motivate it. 
+Features are prioritised according to the impact they may have on the ecosystem, so we appreciate information showing that the impact could be high.
+
+[open an issue]: ./issues/new/choose
```

Comment:
This link and others doesn't work probably due to the relative path. I assume they'd work after merging?